### PR TITLE
Feature/acl

### PR DESCRIPTION
**Pull request by lepsalex**
Add Studies as a new Entity type #105

### CODE REVIEW

**Comment by lepsalex** [comment]
Three remaining Codacy issues are not really issues, two are child classes overwriting a field and the third is a test method - ready for review (will also need to update Readme)

File: docker-compose.yml [hunk 1]
```diff
@@ -29,3 +29,5 @@ services:
       - ./src/main/resources/schemas/01-psql-schema.sql:/docker-entrypoint-initdb.d/init.sql
     expose:
       - "5432"
+    ports:
+      - "8432:5432"
```
Comment (andricDu) [inline docker-compose.yml]: missing newline at end of file

File: pom.xml [hunk 2]
```diff
@@ -95,6 +95,14 @@
             <version>42.1.4</version>
         </dependency>
 
+        <!-- VLAAADDDD -->
```
Comment (andricDu) [inline pom.xml]: :+1:

File: src/main/java/org/overture/ego/model/enums/AclMask.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  @NonNull
+  private final String value;
+
+  @Override
+  public String toString() {
+    return value;
+  }
+
+  public static AclMask fromValue(String value) {
+    for (AclMask aclMask : values()) {
```
Comment (andricDu) [inline src/main/java/org/overture/ego/model/enums/AclMask.java]: val

File: src/main/java/org/overture/ego/repository/AclUserPermissionRepository.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package org.overture.ego.repository;
+
+import org.overture.ego.model.entity.AclUserPermission;
+
+public interface AclUserPermissionRepository
+    extends PermissionRepository<AclUserPermission> {
+}
```
Comment (andricDu) [inline src/main/java/org/overture/ego/repository/AclUserPermissionRepository.java]: new line

File: pom.xml (hunk 2 above)
Comment (lepsalex) [reply pom.xml]: :+1:

File: docker-compose.yml (hunk 1 above)
Comment (lepsalex) [reply docker-compose.yml]: :+1:

File: src/main/java/org/overture/ego/model/enums/AclMask.java (hunk 3 above)
Comment (lepsalex) [reply src/main/java/org/overture/ego/model/enums/AclMask.java]: :+1:

File: src/main/java/org/overture/ego/repository/AclUserPermissionRepository.java (hunk 4 above)
Comment (lepsalex) [reply src/main/java/org/overture/ego/repository/AclUserPermissionRepository.java]: :+1:

**Pull request merged**